artifacts/compatibility-yaml/compatibility-v0.7.0.yaml

```diff
@@ -1,14 +1,22 @@
 {
   "CSI" : {
+    "3.0.0" : {
+      "vSphere": { "min": "6.7.1", "max": "8.2.0" },
```

Comment:
is this validated to work in vsphere 6.7.1

looks like 3.0.0 release carries certain features that only work from 8.0U3 onwards

Reply:
The basic features still works in the older version as well. the combined compatibility with K8s and vsphere will any how not allow to run this on older than 7.x